backend/btrixcloud/crawls.py

```diff
@@ -236,7 +236,7 @@ async def list_crawls(
             query["userid"] = userid
 
         if running_only:
-            query["state"] = {"$in": ["running", "starting", "stopping"]}
+            query["state"] = {"$in": ["running", "starting", "stopping", "waiting"]}
```

Comment:
went to go check if this was in the revised crawl state constants in #837 and saw you beat me to it :)